surveyor/collector_statz.go

```diff
@@ -638,31 +639,53 @@ type connzAggregate struct {
 	msgsRecv  float64
 }
 
-func getConnzAggregate(nc *nats.Conn, account string) (connzAggregate, error) {
+func getConnzAggregate(nc *nats.Conn, numServers int, account string) (connzAggregate, error) {
 	// TODO: Replace with "$SYS.REQ.ACCOUNT.%s.CONNS" after NATS 2.8.4.
 	// CONNS returns bytes sent/recv at the account level without needing the
 	// following code.
 	subj := fmt.Sprintf("$SYS.REQ.ACCOUNT.%s.CONNZ", account)
-	msg, err := nc.Request(subj, nil, 3*time.Second)
+
+	rep := nc.NewRespInbox()
+
+	msg := nats.NewMsg(subj)
+	msg.Reply = rep
+	msg.Data = nil
+
+	s, err := nc.SubscribeSync(msg.Reply)
 	if err != nil {
 		return connzAggregate{}, err
 	}
 
```

Comment:
Prob want to `defer s.Unsubscribe()` here to avoid sub leaks on errors.

Reply:
Alternatively, AutoUnsub on the server count.

Reply:
Oh good call. I just had it at the bottom. I'll move it up.